model.py

```diff
@@ -117,6 +116,7 @@ class GPTConfig:
     block_size: int = 1024
     vocab_size: int = 50304 # GPT-2 vocab_size of 50257, padded up to nearest multiple of 64 for efficiency
     n_layer: int = 12
+    head_size: None | int = None
```

Comment:
It's not working in Python <3.10 - `TypeError: unsupported operand type(s) for |: 'NoneType' and 'type'`. In Python 3.8 I had to use the Union to make it working:
```python
from typing import Union

head_size: Union[None, int] = None
```
Nevertheless I don't think we should create such dependency and moreover we should avoid limiting to use Python 3.10+

Reply:
One more example of "it works on my machine" approach. 
@chrisociepa Thanks for spotting it